src/services/player_database/player_database_service.cpp

```diff
@@ -200,7 +200,14 @@ namespace big
 					{
 						it->second->online_state = PlayerOnlineStatus::OFFLINE;
 						if (online[i] == 1)
+						{
 							it->second->online_state = PlayerOnlineStatus::ONLINE;
+							if (it->second->notify_online)
```

Comment:
This seems to be from my code, I would recommend waiting until my pr gets merged(maybe) then merge master.

Reply:
yes i commented it out left the button